aos/aos_milestones/IMpg/check_impg_run.ipynb

```diff
@@ -0,0 +1,798 @@
+{
```

Comment:
<p>What is the best way to match the wavefront measurements to the image stamps and their locations on the raw images?</p><p>This will be needed frequently for quick debugging.</p><p>Do we have to read it from the Butler, like in the <code>ppg</code>  notebook by Bryce?</p><p>This is probably out of scope for this notebook. Just wondering...</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>This is a great point.. following up on the discussion above, maybe the "raw" wavefront errors could have the position of the donut in the FoV. Both in Ra/Dec and in sensor-pixel coordinates...</p>